### DISCUSSION TITLE
Release videos not working [Jun 18]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#5961] [Discord Threads]
> 🙋‍♂️ **nilsreichardt**  
## The videos in the 0.107 release a not working  
[v.mp4](https://cdn.discordapp.com/attachments/661905455894888493/1109087883891064962/v.mp4)

**Comment by o1lab** [comment]
> 💁 **o1lab**  
#### might be its an issue with github.. will look into it  
#  
> 💁 **o1lab**  
#### updated again,    
####    
#### appreciate notifying about it  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1109087884461486101)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
